openssh-patches/2021-03-02-d2afd717e62d76bb41ab5f3ab4ce6f885c8edc98.patch

```diff
@@ -0,0 +1,594 @@
+From 09eede0a480c65cc006c4ffedd1cc0f15b442d67 Mon Sep 17 00:00:00 2001
+From: Hayden Roche <hayden@wolfssl.com>
+Date: Tue, 31 Aug 2021 13:40:46 -0700
+Subject: [PATCH] Patch for wolfSSL.
+
+This patch was implemented and tested on commit
+d2afd717e62d76bb41ab5f3ab4ce6f885c8edc98. It makes the same changes from
+2020-11-20-9880f3480f9768897f3b8e714d5317fb993bc5b3.patch plus changes to make
+OpenSSH work with wolfSSL FIPS.
```

Comment:
Doesn't work with wolfSSL compiled with `./configure --enable-fips=v2 --enable-openssh --enable-dsa`.

Reply:
Do you have this change? https://github.com/wolfSSL/wolfssl/pull/4352 It should work with DSA on.